src/main/java/org/example/homework11/LettersCounter.java

```diff
@@ -0,0 +1,27 @@
+package org.example.homework11;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LettersCounter {
+    public static void main(String[] args) {
+        Map<Character, Integer> lettersCount = lettersCount("kkkattttttttte");
+        printLetterCount(lettersCount);
+
+    }
+
+    public static Map<Character, Integer> lettersCount(String countedString) {
+        Map<Character, Integer> lettersCount = new HashMap<>();
+        for (int i = 0; i < countedString.length(); i++) {
```

Comment:
```suggestion
        for (char c : countedString.toCharArray()) {
```
you can use toCharArray to simplify your for loop